mail_tracking/models/mail_message.py

```diff
@@ -51,7 +51,8 @@ def tracking_status(self):
                 recipient = (
                     tracking.partner_id.name or tracking.recipient)
                 partner_trackings.append((
-                    status, tracking.id, recipient, tracking.partner_id.id))
+                    status, tracking.id, recipient, tracking.partner_id.id,
+                    tracking.partner_id.email))
```

Comment:
I don't see where are you using this new data in the code below. Could you explain a little bit about how this patch works? :thinking: :sweat_smile:

Reply:
partner_trackings store email and email_cc too, if a partner_tracking without tracking record is present on email_cc it will be marked with "Cc". If an email_cc is currently present on partner_tracking it will not be printed.

Used here: https://github.com/OCA/social/blob/be273331180f017b931231049a097159ca945a6f/mail_tracking/static/src/xml/mail_tracking.xml#L57
And here: https://github.com/OCA/social/blob/be273331180f017b931231049a097159ca945a6f/mail_tracking/static/src/xml/mail_tracking.xml#L83